setup.py

```diff
@@ -43,6 +43,6 @@
     ],
 
     keywords=[],
-    install_requires=['requests', 'PyYAML', 'deprecation', 'pytest', 'mock', 'requests-mock', 'betamax', 'betamax-serializers'],
```

Comment:
perhaps these dependencies should be moved to `tests_require` rather than removed completely